docs/run-keep-ecdsa.adoc

```diff
@@ -0,0 +1,419 @@
+[ethereum.ContractAddresses]
+  BondedECDSAKeepFactory = "0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC"
+
+# Addresses of applications approved by the operator.
+[SanctionedApplications]
+  Addresses = [
+    "0xDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD",
+    "0xEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE"
+  ]
+
+[Storage]
+  DataDir = "/my/secure/location"
+
+[LibP2P]
+  Peers = ["/ip4/127.0.0.1/tcp/3919/ipfs/njOXcNpVTweO3fmX72OTgDX9lfb1AYiiq4BN6Da1tFy9nT3sRT2h1"]
+  Port = 3919
+  # Uncomment to override the node's default addresses announced in the network
+  AnnouncedAddresses = ["/dns4/example.com/tcp/3919", "/ip4/80.70.60.50/tcp/3919"]
+
+[TSS]
+# Timeout for TSS protocol pre-parameters generation. The value
+# should be provided based on resources available on the machine running the client.
+# This is an optional parameter, if not provided timeout for TSS protocol
+# pre-parameters generation will be set to `2 minutes`.
+  PreParamsGenerationTimeout = "2m30s"
+
+```
+
+==== Parameters
+
+[%header,cols=4*]
+|===
+|`ethereum`
+|Description
+|Default
+|Required
+
+|`URL`
+|The Ethereum host your keep-ecdsa will connect to.  Websocket protocol/port.
+|""
+|Yes
+
+|`URLRPC`
+|The Ethereum host your keep-ecdsa will connect to.  RPC protocol/port.
+|""
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`ethereum.account`
+|Description
+|Default
+|Required
+
+|`KeyFile`
+|The local filesystem path to your Keep operator Ethereum account keyfile.
+|""
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`ethereum.ContractAddresses`
+|Description
+|Default
+|Required
+
+|`BondedECDSAKeepFactory`
+|Hex-encoded address of the BondedECDSAKeepFactory Contract.
+|""
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`SanctionedApplications`
+|Description
+|Default
+|Required
+
+|`Addresses`
+|Comma delimited hex-encoded list of application addresses authorized for a given operator.
+|[""]
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`LibP2P`
+|Description
+|Default
+|Required
+
+|`Peers`
+|Comma separated list of network peers to boostrap against.
+|[""]
+|Yes
+
+|`Port`
+|The port to run your instance of Keep on.
+|3919
+|Yes
+
+|`AnnouncedAddresses`
+|Multiaddr formatted hostnames or addresses annouced to the
+Keep Network. More on multiaddr format
+https://docs.libp2p.io/reference/glossary/#multiaddr[in the libp2p
+reference].
+|[""]
+|No
+|===
+
+[%header,cols=4*]
+|===
+|`Storage`
+|Description
+|Default
+|Required
+
+|`DataDir`
+|Location to store the Keep nodes group membership details.
+|""
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`TSS`
+|Description
+|Default
+|Required
+
+|`PreParamsGenerationTimeout`
+|Timeout for TSS protocol pre-parameters generation.
+|"2m"
+|No
+|===
+
+== Build from Source
+
+See the https://github.com/keep-network/keep-core/tree/master/docs/development#building[building] section in our developer docs.
+
+== Docker
+
+=== Get Image
+
+Doesn't exist for the public yet.
+
+=== Run Image
+This is a sample run command for illustration purposes only.
+
+```
```

Comment:
Not sure if this works on .adoc files, but you might be able to add bash after the gate open to get syntax highlighting on the commands within this code block.
```suggestion
``` bash
```

Reply:
It should, though I don’t know if the interstitial space will mess it up.

Reply:
Will give it a shot!

Reply:
This didn't render but 

```
[source,bash]
----
code
----
```

Seems to work.